src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -22,351 +22,358 @@
+            
+        return np.array(image_cropped), np.array(formatted_annotation, dtype=np.float32)
+    
+    return np.array(image), annotation
+
+def complete_augmenter(img_tf, ann_tf, output_height, output_width):
+    img_tf, ann_tf = tf.numpy_function(func=crop_augmenter, inp=[img_tf, ann_tf], Tout=[tf.float32, tf.float32])
+    img_tf, ann_tf = tf.numpy_function(func=padding_augmenter, inp=[img_tf, ann_tf], Tout=[tf.float32, tf.float32])
+    img_tf, ann_tf = tf.numpy_function(func=horizontal_flip_augmenter, inp=[img_tf, ann_tf], Tout=[tf.float32, tf.float32])
+    img_tf, ann_tf = color_augmenter(img_tf, ann_tf)
+    img_tf, ann_tf = hue_augmenter(img_tf, ann_tf)
+    img_tf, ann_tf = resize_augmenter(img_tf, ann_tf, (output_height, output_width))
+    return img_tf, ann_tf
+
+
+
+class DataAugmenter(object):
+    def __init__(self, output_height, output_width, batch_size, resize_only):
+        self.batch_size = batch_size
+        self.graph = tf.Graph()
+        self.resize_only = resize_only
+        with self.graph.as_default():
+            self.img_tf = [tf.placeholder(tf.float32, [None, None, 3]) for x in range(0, self.batch_size )]
+            self.ann_tf = [tf.placeholder(tf.float32, [None, 6]) for x in range(0, self.batch_size )]
+            self.resize_op_batch = []
+            for i in range(0, self.batch_size):
+                if resize_only:
+                    aug_img_tf, aug_ann_tf = resize_augmenter(self.img_tf[i], self.ann_tf[i], (output_height, output_width))
+                    self.resize_op_batch.append([aug_img_tf, aug_ann_tf])
+                else:
+                    aug_img_tf, aug_ann_tf = complete_augmenter(self.img_tf[i], self.ann_tf[i], output_height, output_width)
+                    self.resize_op_batch.append([aug_img_tf, aug_ann_tf])
+
+    def get_augmented_data(self, images, annotations):
+        with tf.Session(graph=self.graph) as session:
+            feed_dict = dict()
+            graph_op = self.resize_op_batch[0:len(images)]
+            for i in range(0, len(images)):
+                feed_dict[self.img_tf[i]] = _utils.convert_shared_float_array_to_numpy(images[i])
+                if self.resize_only:
+                    feed_dict[self.ann_tf[i]] = self.batch_size * [np.zeros(6)]
+                else:
+                    feed_dict[self.ann_tf[i]] = _utils.convert_shared_float_array_to_numpy(annotations[i])
+            aug_output = session.run(graph_op, feed_dict=feed_dict)
+            processed_images = []
+            processed_annotations = []
+            for o in aug_output:
+                processed_images.append(o[0])
+                processed_annotations.append(np.ascontiguousarray(o[1], dtype=np.float32))
+            processed_images = np.array(processed_images, dtype=np.float32)
+            processed_images = np.ascontiguousarray(processed_images, dtype=np.float32)
+            return tuple((processed_images, processed_annotations))
```

Comment:
`return (processed_images, processed_annotations)` will do the same thing. Calling `tuple` is unnecessary.

Reply:
Yeah will change it.